go/teams/errors.go

```diff
@@ -552,3 +552,27 @@ func NewMissingReaderKeyMaskError(gen keybase1.PerTeamKeyGeneration, app keybase
 func (e MissingReaderKeyMaskError) Error() string {
 	return fmt.Sprintf("missing reader key mask for gen:%v app:%v", e.gen, e.app)
 }
+
+type MapAncestorsError struct {
+	err error
+
+	// failedLoadingAtAncestorIdx indicates the team the MapTeamAncestors load failed at.
+	// So if it failed loading the initial team passed in, failedLoadingAtAncestorIdx=0.
+	// Each successive parent team adds one to the index. So if the team tree were
+	// A.B.C.D.E and MapTeamAncestors was called on D, and the function failed at B,
+	// failed=2.
+	failedLoadingAtAncestorIdx int
```

Comment:
I wonder if it's possible to get the name of the team which ancestor failed to load in there

Reply:
but you probably tried it since you came up with this system, I wonder what's the reason why this is hard. I haven't tried it myself

Reply:
+1 was thinking the same

Reply:
maybe. we could optionally pass in the teamname to this function, seems messy either way though. i agree this arithmetic just for getting the team name is silly though